doc/explanation/external-builder.md

```diff
@@ -0,0 +1,5 @@
+# External builder
+
+In order to pre-bootstrap LXD, Microk8s snaps to Juju, external VMs are spanwed and then run with
```

Comment:
Nitpicks: Microk8s --> MicroK8s, spanwed --> spawned 

Clarifying questions: Does MicroK8s spawn the external VMs? I'm trying to figure out who's doing the running here: Does MicroK8s run, or do the external VMs run?

Reply:
The external VMs are run on OpenStack. The VM contains bootstrapped MicroK8s and LXD with Juju which facilitates integration testing for many teams :)

Reply:
Thank you for the explanation, I understand now :) I suggest the following changes to this sentence (you could also replace the semicolon with a period and split into two sentences. I just want to make it clear that the external VMs are doing the running).

> In order to pre-bootstrap LXD, MicroK8s snaps to Juju; then, external VMs are spawned and ran with cloud-init script that installs the required components.